p2p/src/main/java/bisq/network/p2p/P2PModule.java

```diff
@@ -105,5 +105,6 @@ protected void configure() {
         bindConstant().annotatedWith(named(NetworkOptionKeys.MSG_THROTTLE_PER_10_SEC)).to(environment.getRequiredProperty(NetworkOptionKeys.MSG_THROTTLE_PER_10_SEC));
         bindConstant().annotatedWith(named(NetworkOptionKeys.SEND_MSG_THROTTLE_TRIGGER)).to(environment.getRequiredProperty(NetworkOptionKeys.SEND_MSG_THROTTLE_TRIGGER));
         bindConstant().annotatedWith(named(NetworkOptionKeys.SEND_MSG_THROTTLE_SLEEP)).to(environment.getRequiredProperty(NetworkOptionKeys.SEND_MSG_THROTTLE_SLEEP));
+        bindConstant().annotatedWith(named("MAX_SEQUENCE_NUMBER_MAP_SIZE_BEFORE_PURGE")).to(10000);
```

Comment:
hasn't that been only 1000 (one thousand) previously? I guess the memory impact is marginal anyways but can you elaborate your decision?

Reply:
Yes, this was intended to just be a refactor, not a policy change. An extra 0 was added accidentally during the move. Thanks for the keen eye.